doc/configuration.rst

```diff
@@ -25,7 +25,7 @@ Top-level configuration
 
 .. literalinclude:: ../config.default.yaml
    :language: yaml
-   :lines: 5-12,20,31-38
+   :lines: 5-12,24,31-38
```

Comment:
I think we will change that by the `start-at` and `end-at` stuff?

Reply:
yes @pz-max, that is the plan. Just did this to motivate the `start-at` and `end-at` approach. Will work through this soon I hope, a lot of work has popped up recently.